main.js

```diff
@@ -0,0 +1,414 @@
+    if (playerOName === "") {
+      playerOName = "Player O";
+    }
+    //changes display for Player O to show chosen name
+    playerONameDisplay.textContent = playerOName;
+    //resets the player O name form
+    setPlayerOName.reset();
+  });
+}
+
+// Player vs Player game function
+function playGame(event) {
+  //guard clause to prevent a previously clicked cell from being clicked again. If it has already been clicked the player is alerted to pick an empty cell.  Otherwise it marks the cell appropriately
+  if (event.target.textContent !== "") {
+    alert("Please select an empty cell.");
+  }
+  // when the textContent of the cell will change to X or O depending on the currentPlayer.  The current player is then changed
+  else if (event.target.textContent === "") {
+    if (currentPlayer === "X") {
+      event.target.textContent = "X";
+    } else {
+      event.target.textContent = "O";
+    }
+
+    //check for win condition by calling win() function
+    //if win returns true, the current player is declared the winner, the start button is re-enabled and playGame click listener is removed from all cells
+    if (win()) {
+      playerStatus.textContent = `${currentPlayerName} Won!`;
+      start.disabled = true;
+      playerVplayer.disabled = false;
+      playerVcomputer.disabled = false;
+
+      // removes event listener on cells
+      for (let elements of cells) {
+        elements.removeEventListener("click", playGame);
+      }
+      //resets timer
+      clearInterval(timer);
+      
+      //check for tie condition
+      //if tie, re-enable buttons to play again
+    } else if (tie()) {
+      playerStatus.textContent = " It's a Tie, Play Again?";
+      start.disabled = true;
+      playerVplayer.disabled = false;
+      playerVcomputer.disabled = false;
+      for (let elements of cells) {
+        elements.removeEventListener("click", playGame);
+      }
+      //resets timer
+      clearInterval(timer);
+
+      // changes players turn while win and tie functions are false
+    } else {
+      if (currentPlayer === "X") {
+        currentPlayer = "O";
+        currentPlayerName = playerOName;
+      } else {
+        currentPlayer = "X";
+        currentPlayerName = playerXName;
+      }
+      playerStatus.textContent = `${currentPlayerName}'s Turn`;
+    }
+  }
+}
+
+
+// Player vs Computer game function
+function playComputerGame(event) {
+  if (event.target.textContent !== "") {
+    alert("Please select an empty cell.");
+  } else if (event.target.textContent === "") {
+
+    // when the textContent of the cell will change to X or O depending on the currentPlayer.  The current player is then changed
+    if (currentPlayer === "X") {
+      event.target.textContent = "X";
+    } else {
+
+      // random guess by computer via randomGuess function
+      let randomGuess = Math.floor(Math.random() * 8);
+      while (cells[randomGuess].textContent !== "") {
+        randomGuess = Math.floor(Math.random() * 8);
```

Comment:
You can turn this into a function, and look up the `.click()` method. Here is some information on that. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click